modules/context/repo.go

```diff
@@ -554,5 +554,6 @@ func UnitTypes() macaron.Handler {
 		ctx.Data["UnitTypeSettings"] = models.UnitTypeSettings
 		ctx.Data["UnitTypeExternalWiki"] = models.UnitTypeExternalWiki
 		ctx.Data["UnitTypeExternalTracker"] = models.UnitTypeExternalTracker
+		ctx.Data["UnitTypeBranches"] = models.UnitTypeBranches
```

Comment:
I don't think new unit type is needed as you don't use it anyway. I think unit type code that you are using already is enough.